Task3.py

```diff
@@ -0,0 +1,19 @@
+# Multi-paradigm programming languages, Task №3
```

Comment:
![image](https://user-images.githubusercontent.com/43804299/224623056-b84edfe6-15cf-4afe-bc65-e33cc4996570.png)
you skipped this point

Reply:
please consider comments from the previous PR